### DISCUSSION TITLE
Trouble with Video setup

### DISCUSSION THREAD

**Discussion by Fullyfly** [iNavFlight/inav#10515] [Support]
Hi All, Been away from RC for several years and just getting back into it making a Fishing Quad. Have got all flying perfect on INAV all GPS functions working great. Just cant get my FPV Camera going.
I have a Speedybee flight controller, Foxeer Camera and Zeus VTX, Fatshark Dominator 1 goggles with built in VRX.  I have never setup FPV cameras through a flight controller before. I am mainly just getting snow but some channels and setting I get a solid black screen like its getting some sort of signal.
So understand I need to assign a UART. Have tried just about every video option in UART 3 and 5 as the 2 available. Not sure how you know which one you are connected to? 
On the OSD page I have tried both NTSC and PAL. Have scrolled through every channel on my VTX and Goggles and nothing but snow or a black screen. 
Just feel there is something im missing with the INAV setup.
I got a new zeus mini VTX for my mini quad which is connected direct to camera and video works great. Just cant get it going on my new quad through the speedybee and INAV. 
Any advice greatly appreciated. 
Cheers

**Comment by anti-vaxxer** [comment]
Hi Fullyfly,
can you show photos of your FC wiring?
camera video wire goes to CAM pad on FC and VTX video wire to VTX pad on FC.
VTX control wire goes to a UART TX on FC.

**Comment by sensei-hacker** [comment]
A black screen MAY indicate your VTX is working on that channel or a nearby channel, but there's a camera issue. Of course I've never left a lens cap on on my 1200mm plane a few months ago, but I heard that has happened. :)

Sometimes when things are troublesome it's helpful to start by first skipping the flight controller and connecting the camera directly to the VTX. Set the channel and power level using the buttons on the VTX and VRX. Do NOT "auto detect" the channel - that will often find a nearby channel, but the wrong channel. Simply connecting the camera to the VTX, without any flight controller stuff in between, eliminates any possibility of problems related to either the OSD on the flight controller or the SmartAudio control via the flight controller (the uart).

AFTER you have it working without any flight controller in sight, THEN you can route the video wire through the flight controller for OSD. The wire from the camera goes to the FC pad marked for the camera, or "video in". The video wire to the VTX goes to the one marked VTX or "video out". If you have trouble at that point let us know and please provide photos of exactly how you have it wired.

After you have OSD working, THEN think about connecting the SmartAudio wire to a uart to be able to have the power level controlled by the flight controller. One step at a time.

Right now, you're trying to get six different things working together at once - the camera, the OSD, the VTX, the VRX, the goggles, the SmartAudio ... and you have no idea where the problem is.
It'll be simpler to eliminate half the stuff for a moment and just get the VTX talking to the VRX - get them on the same channel etc.  Then connect the camera to the VTX without mixing in all of that other stuff right away.

**Comment by Fullyfly** [comment]
Hi Everyone. Thanks for the replies. I took a pic of the flight controller and working backwards I worked it out. I must not have had the correct plug in the VTX and the video wire was red. The standard yellow one on the RX port. It went V - G - RX and the RX was yellow. I must have just thought that was video RX but its the RX for the flight control. Anyway swapped the wire and now have a picture. Ran out of time to do any more. No OSD (Other than the Foxeer) showing but will work on that tomorrow. :-)
Picture of wiring below. The T1 port on the Speedybee goes to the RX on the Zeus yeah ? Hopefully just some setup from there.
![1733727690957](https://github.com/user-attachments/assets/dfd2f21a-2816-427d-94e4-646d948b5f64)
Cheers

**Reply by anti-vaxxer** [reply]
yes T1 port on the Speedybee goes to the RX on the Zeus

**Reply by Fullyfly** [reply]
Great ill put it all back together. Rest should be in settings. 
I don't fully understand the UART allocation. Is that T1 pad a fixed port or do you just tell the device what port it is by selecting the device on any available port? Not sure which one to choose, Just runcam or one of the generic ones ?
So then need to setup a user flight mode to turn it on ? Is it defaulted to off ?
Thanks for the help.
Cheers

**Reply by anti-vaxxer** [reply]
In Ports tab for UART1 Peripherals select IRC Tramp
![image](https://github.com/user-attachments/assets/6eab53dd-8ae9-4c40-9145-7a891920bd6d)

>  So then need to setup a user flight mode to turn it on ? Is it defaulted to off ?

what do you mean? the VTX is always powered when you have battery connected. you can change band, channel, power level (pit mode would be another setting but it doesn't work in INAV) in INAV Configurator from Configuration tab or OSD menu with your radio sticks.

**Reply by sensei-hacker** [reply]
>  ?So then need to setup a user flight mode to turn it on ? Is it defaulted to off ?


Some flight controllers have the VTX power  pad/plug turned off by default and you turn it on the same way you switch flight modes. Using USER1 or USER2 on the Modes tab.
Check the documentation of your specific flight controller.  Or use a multimeter to test for voltage if it's not already on.
